python/frontend/cirrus/cirrus/parameter_server.py

```diff
@@ -87,7 +87,7 @@ def start(self, config):
             "nohup",
             "./parameter_server",
             "--config", config_filename,
-            "--nworkers", str(self._num_workers),
+            "--nworkers", str(self._num_workers + 20),
```

Comment:
What is this 20?

Reply:
Messenger.py needs additional connections in order to connect to the parameter server. Without the 20, the parameter server would reject any connections made to it since it only allows self._num_workers number of concurrent connections.